pkg/scaffold/helm/go_mod.go

```diff
@@ -0,0 +1,115 @@
+	github.com/coreos/prometheus-operator v0.26.0 // indirect
+	github.com/dgrijalva/jwt-go v3.2.0+incompatible // indirect
+	github.com/emicklei/go-restful v2.8.1+incompatible // indirect
+	github.com/go-logr/logr v0.1.0 // indirect
+	github.com/go-logr/zapr v0.1.0 // indirect
+	github.com/go-openapi/spec v0.18.0 // indirect
+	github.com/golang/groupcache v0.0.0-20180924190550-6f2cf27854a4 // indirect
+	github.com/google/btree v0.0.0-20180813153112-4030bb1f1f0c // indirect
+	github.com/google/gofuzz v0.0.0-20170612174753-24818f796faf // indirect
+	github.com/google/uuid v1.0.0 // indirect
+	github.com/googleapis/gnostic v0.2.0 // indirect
+	github.com/gophercloud/gophercloud v0.0.0-20190318015731-ff9851476e98 // indirect
+	github.com/gregjones/httpcache v0.0.0-20180305231024-9cad4c3443a7 // indirect
+	github.com/imdario/mergo v0.3.6 // indirect
+	github.com/json-iterator/go v1.1.5 // indirect
+	github.com/kr/pretty v0.1.0 // indirect
+	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
+	github.com/modern-go/reflect2 v0.0.0-20180701023420-4b7aa43c6742 // indirect
+	github.com/onsi/ginkgo v1.7.0 // indirect
+	github.com/onsi/gomega v1.4.3 // indirect
+	github.com/operator-framework/operator-sdk v0.6.0
+	github.com/pborman/uuid v0.0.0-20180906182336-adf5a7427709 // indirect
+	github.com/peterbourgon/diskv v2.0.1+incompatible // indirect
+	github.com/spf13/pflag v1.0.3
+	go.opencensus.io v0.19.2 // indirect
+	go.uber.org/atomic v1.3.2 // indirect
+	go.uber.org/multierr v1.1.0 // indirect
+	go.uber.org/zap v1.9.1 // indirect
+	golang.org/x/time v0.0.0-20180412165947-fbb02b2291d2 // indirect
+	golang.org/x/tools v0.0.0-20190124215303-cc6a436ffe6b // indirect
+	gopkg.in/inf.v0 v0.9.1 // indirect
+	k8s.io/api v0.0.0-20181126151915-b503174bad59 // indirect
+	k8s.io/apiextensions-apiserver v0.0.0-20181126155829-0cd23ebeb688 // indirect
+	k8s.io/apimachinery v0.0.0-20181126123746-eddba98df674
+	k8s.io/client-go v2.0.0-alpha.0.0.20181126152608-d082d5923d3c+incompatible
+	k8s.io/code-generator v0.0.0-20180823001027-3dcf91f64f63
+	k8s.io/gengo v0.0.0-20181113154421-fd15ee9cc2f7 // indirect
+	k8s.io/klog v0.1.0 // indirect
+	k8s.io/kube-openapi v0.0.0-20180711000925-0cf8f7e6ed1d
+	sigs.k8s.io/controller-runtime v0.1.10
+	sigs.k8s.io/testing_frameworks v0.1.0 // indirect
+	sigs.k8s.io/yaml v1.1.0 // indirect
+)
+
+replace (
+	// The following are locked to kubernetes-1.13.3
+	k8s.io/api => k8s.io/api v0.0.0-20190202010724-74b699b93c15
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.0.0-20190202013456-d4288ab64945
+	k8s.io/apimachinery => k8s.io/apimachinery v0.0.0-20190117220443-572dfc7bdfcb
+	k8s.io/client-go => k8s.io/client-go v2.0.0-alpha.0.0.20190202011228-6e4752048fde+incompatible
+	k8s.io/code-generator => k8s.io/code-generator v0.0.0-20181117043124-c2090bec4d9b
+	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20180711000925-0cf8f7e6ed1d
+)
+`
```

Comment:
I'm not able to get this `go.mod` file to work out of the box:

```
/home/joe/go/pkg/mod/k8s.io/kubernetes@v1.14.0/pkg/printers/internalversion/printers.go:33:2: unknown import path "k8s.io/api/coordination/v1": cannot find module providing package k8s.io/api/coordination/v1
/home/joe/go/pkg/mod/k8s.io/kubernetes@v1.14.0/pkg/apis/scheduling/v1/register.go:20:2: unknown import path "k8s.io/api/scheduling/v1": cannot find module providing package k8s.io/api/scheduling/v1
/home/joe/go/pkg/mod/k8s.io/helm@v2.13.1+incompatible/pkg/kube/client.go:49:2: unknown import path "k8s.io/cli-runtime/pkg/genericclioptions/resource": cannot find module providing package k8s.io/cli-runtime/pkg/genericclioptions/resource
/home/joe/go/pkg/mod/k8s.io/cli-runtime@v0.0.0-20190313123343-44a48934c135/pkg/genericclioptions/config_flags.go:31:2: unknown import path "k8s.io/client-go/discovery/cached/disk": cannot find module providing package k8s.io/client-go/discovery/cached/disk
```

It looks like we need to declare a few more dependencies.

I was able to get things working by scaffolding the hybrid helm operator `Gopkg.toml` and `Gopkg.lock` files from `master` and then running `go mod init && go mod tidy`. Should we use that resulting `go.mod` file as the initial template?

Reply:
Yes I'll go ahead and update both `ansible` and `helm` `go_mod.go` files.